migrations/sql/V2020.02.04_1438__DocumentSchema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE "orderlyweb_document_dir" (
+    "path" TEXT,
+    "parent" TEXT,
+    PRIMARY KEY ("path"),
```

Comment:
I'm just thinking about how we'll display these in the index page.
If we might want to display them like:

```
model_review
       yellow_fever
             file1.csv
             file2.doc
```
etc, then might be more useful to have `directory_name` rather than `path` here? The path can always be created by concatenating parent names, or we could store both. I reckon we want `display_name` and `show` for directories as well.

Reply:
That's true! Maybe we could keep the path as the primary key, since that ought to be unique, and I'll add in directory_name, display_name and show?